app/js/controllers/main.js

```diff
@@ -9,16 +8,23 @@ import DevController from './dev';
 
 import Foxbox from '../lib/foxbox/foxbox';
 
-export default class MainController extends RoutingController {
+const p = Object.freeze({
```

Comment:
We used routing controller only in one place and we're going to migrate to some other framework, so here again I tried to migrate only things that we use and just merged RoutingController with MainController. If in the future we need it, it's pretty easy to extract it.